src/styles/navbar.css

```diff
@@ -0,0 +1,72 @@
+/* Search bar styles */
+.search-input {
+  flex-basis: 40%;
+}
+
+.search-input input {
+  padding: var(--size-5) var(--size-9) var(--size-5) var(--size-17);
+}
+
+.search-input .search-icon {
+  color: var(--color-grey);
+  font-size: var(--size-11);
+}
+
+.hide-search {
+  display: none;
+}
+
+/* Mobile search bar styles */
+.mob-search-input {
+  position: sticky;
+  top: 4rem;
+  z-index: 9999;
+  width: 100%;
+  background-color: var(--primary-extra-light-color);
+  margin-top: var(--size-5);
+}
+
+.mob-search-input input {
+  display: block;
+  width: 100%;
+  margin: 0 auto;
+  padding: var(--size-5) var(--size-9);
+}
+
+/* Navbar styles */
+.nav-item:nth-child(n) {
+  margin: 0 var(--size-9);
+}
+
+.nav-logo {
+  margin: 0 var(--size-5);
+}
+
+/* Navbar icon badges styles */
+.top-right-icon {
+  top: 0;
+  right: 0;
+  transform: translate(25%, -50%);
+}
+
+/* Media queries for tablets and up */
+@media screen and (max-width: 768px) {
```

Comment:
use of relative units is good practice.

Reply:
That is true but I would prefer to use pixels here just so it is easier to know which screen size I am targeting directly, and also to me makes this media query easy to read